lib/core/card-view/card-view.module.ts

```diff
@@ -18,47 +18,76 @@
 import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
 import { FormsModule } from '@angular/forms';
-import { MatButtonModule, MatDatepickerModule, MatIconModule, MatInputModule, MatNativeDateModule } from '@angular/material';
+import {
+    MatButtonModule,
+    MatDatepickerModule,
+    MatIconModule,
+    MatInputModule,
+    MatCheckboxModule,
+    MatNativeDateModule
+} from '@angular/material';
+import { MatDatetimepickerModule, MatNativeDatetimeModule } from '@mat-datetimepicker/core';
 import { FlexLayoutModule } from '@angular/flex-layout';
 import { TranslateModule } from '@ngx-translate/core';
 
-import { CardViewContentProxyDirective } from './card-view-content-proxy.directive';
-import { CardViewDateItemComponent } from './card-view-dateitem.component';
-import { CardViewItemDispatcherComponent } from './card-view-item-dispatcher.component';
-import { CardViewMapItemComponent } from './card-view-mapitem.component';
-import { CardViewTextItemComponent } from './card-view-textitem.component';
-import { CardViewComponent } from './card-view.component';
+import { CardViewContentProxyDirective } from './directives/card-view-content-proxy.directive';
+import {
+    CardViewBoolItemComponent,
+    CardViewDateItemComponent,
+    CardViewItemDispatcherComponent,
+    CardViewMapItemComponent,
+    CardViewTextItemComponent,
+    CardViewComponent
+} from './components/card-view.components';
+
+import { CardItemTypeService } from './services/card-item-types.service';
+import { CardViewUpdateService } from './services/card-view-update.service';
+
+const MATERIAL_MODULES = [
```

Comment:
can we remove those array in the past we had a mix of array and normal import and was really difficult to maintain